csv_detective/detect_fields/FR/geo/region/__init__.py

```diff
@@ -1,8 +1,48 @@
-from frformat import Region
+from frformat import Region, Options, Millesime
 
 PROPORTION = 1
 
+_extra_valid_values_set = frozenset({
```

Comment:
So this has to stay despite
```
_options = Options(
    ignore_case=True,
    ignore_accents=True,
    replace_non_alphanumeric_with_space=True,
    ignore_extra_whitespace=True,
)
```
? Why for this this specific case and not the others? 🤔

Reply:
The extra valid values are here to reproduce the former behaviour (before introduction of fr-format). csv-detective had a lot of values that are not proper regions (abbreviations like bfc, aura, as well as former regions). Other formats did not have such a variety of extra values.